code/modules/projectiles/guns/projectile/pistol/colt.dm

```diff
@@ -46,3 +46,11 @@
 	grip = /obj/item/part/gun/grip/wood
 	mechanism = /obj/item/part/gun/mechanism/pistol
 	barrel = /obj/item/part/gun/barrel/pistol
+
+/obj/item/gun/projectile/colt/relic
+	name = "Colt \"M1911\" Pistol"
```

Comment:
While real life gun manufacturers do exist in our lore, I don't think it's a good idea to put any of them in the gun name.

Reply:
with the clear "This is a copy of x weapon by y real life company" in both these weapon's descs, it's hard to make an authentic old-world relic version without basically saying that it's said weapons.